cli/manage.go

```diff
@@ -244,8 +244,8 @@ func manage(c *cli.Context) {
 
 	refreshMinInterval := c.Duration("engine-refresh-min-interval")
 	refreshMaxInterval := c.Duration("engine-refresh-max-interval")
-	if refreshMinInterval == time.Duration(0)*time.Second {
-		log.Fatal("minimum refresh interval should be a positive number")
+	if refreshMinInterval <= time.Duration(0)*time.Second || refreshMaxInterval <= time.Duration(0)*time.Second {
```

Comment:
I think just using `refreshMinInterval <= time.Duration(0)*time.Second` here is OK